benchmarks/dynamo/_onnx/patch.py

```diff
@@ -0,0 +1,49 @@
+from torch.utils import _pytree as pytree
+
+
+def patch_non_tensor_outputs(correct_result, new_result, fp64_outputs):
+    try:
+        from transformers import modeling_outputs
+    except ImportError:
+        has_transformers = False
+    else:
+        has_transformers = True
```

Comment:
Not following the question on discrepancy.

Are you worried that benchmark runs with huggingface suite, yet this import fails? We will get accuracy check failed for these models if that happens.

Reply:
Yes, I was worried that the accuracy would change depending on `has_transformers` value.
Which brings the question whether we should make `transformers` mandatory to have uniform results.

Imagine that for 6 months `transformers` is not installed and we log one result. Next, we install it on the environment and the accuracy jumps (for bad and good). If that happens (my original question was if that is a possibility), the accuracy discrepancy would be because of environment change, not any improvement or regression on the exporter.

Reply:
I see, in that case that's fine, because if `has_transformers` is False, it means no transformers models were tested, so that part of patch is irrelevant.